idem_azurerm/states/azurerm/postgresql/server_security_alert_policy.py

```diff
@@ -273,13 +273,13 @@ async def present(
         ret["result"] = True
         ret[
             "comment"
-        ] = f"The server security alert policy for the server {name} has been {action}d."
+        ] = f"The server security alert policy {name} for the server {server_name} has been {action}d."
```

Comment:
This is not consistent with the other string format statements. Any reason?

Reply:
I am not sure why that line was written in an inconsistent manner. However, it is inconsistent like that in every single state.

Reply:
Ok good call! Thanks